apps/condo/domains/user/components/containers/styles.tsx

```diff
@@ -69,6 +69,13 @@ export const PageContent = styled.div<IContentProps>`
   overflow-y: scroll;
 `
 
+export const ReCaptchaContainer = styled.div`
+  position: absolute;
+  visibility: hidden;
+  bottom: 0;
+  z-index: 1000;
```

Comment:
Seems to me z-index property isn't necessary here, because this element is hidden. Also you can try remove position and bottom properties